### ISSUE TITLE
Can't locate object method "new" via package "Tags::HTML::Pager"

### ISSUE THREAD

**Issue by eserte** [michal-josef-spacek/Tags-HTML-Pager#2]
t/Tags-HTML-Pager/03-new.t and t/Tags-HTML-Pager/05-process_css.t fail with the following error
```
Can't locate object method "new" via package "Tags::HTML::Pager" at /home/cpansand/.cpan/build/2022091919/Tags-HTML-Pager-0.01-uzTrH1/blib/lib/Tags/HTML/Pager.pm line 24.
# Looks like your test exited with 255 just after 1.
t/Tags-HTML-Pager/03-new.t .......... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 6/7 subtests 
```

**Comment by michal-josef-spacek** [comment]
Hi,
Fixed in 0.02 release.
Thanks.

**Comment by michal-josef-spacek** [comment]
Ha, sorry, another bug. Release in 0.03
